src/index.js

```diff
@@ -1 +1,33 @@
-import './style.css';
\ No newline at end of file
+/* eslint-disable no-plusplus */
```

Comment:
- [ ] Please avoid disabling linter rules! They serve as a way of following best practices and maintaining a consistent code format. Instead, consider solving the issues! 🤓 🔧